src/include/txn_inline.h

```diff
@@ -909,6 +910,16 @@ __wt_upd_alloc(WT_SESSION_IMPL *session, const WT_ITEM *value, u_int modify_type
         (value != NULL &&
           !(modify_type == WT_UPDATE_RESERVE || modify_type == WT_UPDATE_TOMBSTONE)));
 
+    /*
+     * Ensure that the memory allocated is never less than the size reported by sizeof. Otherwise
+     * bit-exact tools like Google MSan may infer the structure is not completely initialized.
+     */
+    if (value != NULL) {
```

Comment:
There is no need for `{}` because of a single statement. Same for the else condition.